### PR TITLE
Added latest Girl Month and the ability to extend into next year

### PR DESCRIPTION
**Pull request by philiplawall**
I think this should work :)

And F1nn F1nn F1nn I had to extend it into next Year ...

### CODE REVIEW

File: calendar.js [hunk 1]
```diff
@@ -14,32 +14,41 @@ const dayTemplate = document.getElementById("day-template").content
 const generateMonths = () => {
   const main = document.getElementById("main")
 
+  let year = now.getFullYear()
+  let yearMonth = monthRange[0]
```
Comment (jenchanws) [inline calendar.js]:
```suggestion
```

File: calendar.js [hunk 2]
```diff
@@ -14,32 +14,41 @@ const dayTemplate = document.getElementById("day-template").content
...
+
   for (let month = monthRange[0]; month <= monthRange[1]; month++) {
-    let monthStart = new Date(now.getFullYear(), month - 1, 1)
+    let monthStart = new Date(year, yearMonth - 1, 1)
```
Comment (jenchanws) [inline calendar.js]:
```suggestion
    let monthStart = new Date(year, month - 1, 1)
```

File: calendar.js [hunk 3]
```diff
@@ -14,32 +14,41 @@ const dayTemplate = document.getElementById("day-template").content
...
     let monthName = monthFormatter.format(monthStart)
 
     let monthDiv = monthTemplate.cloneNode(true)
     let monthNameHeader = monthDiv.querySelector("h3")
     monthNameHeader.textContent = monthName
-    generateDays(month, monthDiv.querySelector("ol"))
+    generateDays(year, yearMonth, monthDiv.querySelector("ol"))
```
Comment (jenchanws) [inline calendar.js]:
```suggestion
    generateDays(monthStart.getFullYear(), monthStart.getMonth() + 1,
      monthDiv.querySelector("ol"))
```

File: calendar.js [hunk 4]
```diff
@@ -14,32 +14,41 @@ const dayTemplate = document.getElementById("day-template").content
...
 
     if (monthStart.getDay() >= 2)
       monthNameHeader.style = "margin-top: 60px; margin-bottom: -60px"
     main.appendChild(monthDiv)
+    if (month % 12 == 0){
+      year++  
+      yearMonth = 1
+    }else{
+      yearMonth++
+    }
```
Comment (jenchanws) [inline calendar.js]:
```suggestion
```

**Review by jenchanws — CHANGES_REQUESTED** [review]

File: calendar.js [hunk 5]
```diff
@@ -62,8 +65,9 @@ const populateCalendar = () => {
     for (let d = 0; d < goal.days; d++) {
       const date = new Date(year, month - 1, day + d)
 
-      let [mo, dy] = [date.getMonth() + 1, date.getDate()]
-      let dayDiv = document.getElementById(`day-${mo}-${dy}`)
+      let [yr, mo, dy] = [date.getFullYear(), date.getMonth() + 1, date.getDate()]
+      console.log(`day-${yr}-${mo}-${dy}`)
```
Comment (jenchanws) [inline calendar.js]:
```suggestion
```

**Review by jenchanws — APPROVED** [review]

File: calendar.js (hunk 5 above)
Comment (philiplawall) [reply calendar.js]: whoops

**Pull request merged**